src/store/modules/user/actions.ts

```diff
@@ -111,14 +115,20 @@ const actions: ActionTree<UserState, RootState> = {
     commit(types.USER_INSTANCE_URL_UPDATED, payload)
   },
 
-  async getShopifyConfig({ commit }) {
-    const resp = await UserService.getShopifyConfig({
-      "entityName": "ShopifyConfig",
-      "noConditionFind": "Y"
-    })
+  async getShopifyConfig({ commit }, productStoreId) {
+    if (productStoreId) { 
```

Comment:
I think we should handle case when either productStoreId is not passed or response returned empty or no record by resetting the current config

Reply:
Updated the code.